ui-charts/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import '@osrd-project/ui-core/dist/theme.css';
+
+export * from './spaceTimeChart';
+
+export * from './manchette';
+
+export { SpeedSpaceChart } from './speedSpaceChart';
+
+export { TrackOccupancyCanvas, TrackOccupancyManchette } from './trackOccupancyDiagram';
```

Comment:
Could we `export *` for these last 2, to make sure we don't miss anything when adding new stuff to exports in subdirs?